apps/docs/package.json

```diff
@@ -26,7 +26,8 @@
     "preview": "qwik build preview && vite preview --open",
     "serve": "wrangler pages dev ./dist --compatibility-flags=nodejs_als",
     "start": "vite --open --mode ssr",
-    "qwik": "qwik"
```

Comment:
This won't work... We need the `qwik` command. Please visit https://qwik.dev/ for more information